readme.md

```diff
@@ -26,10 +26,10 @@ The plugin streamlines the entire configuration process and removes the need for
 
 ## Requirements
 
-- Jenkins 2.222.4+
+- Jenkins 2.204.6+
```

Comment:
actually, this is not right, we require 2.222 don't we?

Reply:
We did. This is correct on master, the branch was stale, so I've removed this change.